### ISSUE TITLE
chip-seq pipeline

### ISSUE THREAD

**Issue by yong27** [chapmanb/bcbio-nextgen#1200]
Thanks for great workflow program!

I tried chip-seq pipeline like follows.

    $ bcbio_nextgen.py -w template illumina-chipseq project1 my.fastq
    $ bcbio_nextgen.py config/project1.yaml -n 4

But it makes following errors.

    ValueError: Unexpected configuration keyword in 'algorithm' section: ['peakcaller']
    See configuration documentation for supported options:
    https://bcbio-nextgen.readthedocs.org/en/latest/contents/configuration.html#algorithm-parameters

Do I have to set for "peakcaller" algorithm? How can I do that?

**Comment by lpantano** [comment]
Hi,

sorry about this. Chipseq only will work with the development version. You will need to update to bcbio_nextgen.py upgrade -u development —tools

Let me know if this fix the problem, or you already have the development version, and I will try to debug further.

cheers

**Comment by lpantano** [comment]
I will close now, but feel free to re-open if you find more issues.
thanks!

**Comment by mpschr** [comment]
Hi

Is this still the case? I got the error after a "standard" upgrade. Do I need to switch to development code?

**Comment by lpantano** [comment]
Hi,

At this point is part of the stable version as well. So any upgrade to stable will be able to run chipseq now.

cheers

**Comment by mpschr** [comment]
I still get the error after performing `bcbio_nextgen.py upgrade -u stable --tools`. But also the bcbio_python version does not seem to change: it stays at `0.9.1a`

**Comment by mpschr** [comment]
Ok, I solved my problem - as a little summary for other people:

Conda 4.2.x creates a lock and fails on this very lock when installing the gffutils 0.8.4rc1 from the bcbio channel. I downloaded the package and installed manually, but conda does not detect that it is from the bcbio channel and tries to install it again, failing on its own lock. To solve this, temporally comment the line where the dependencies are installed `subprocess.check_call([conda_bin, "install"....`. After this everything went fine and the changed script was replaced in any case
